addons/web/static/src/views/form/button_box/button_box.scss

```diff
@@ -105,15 +105,20 @@
         margin-left: -$border-width;
     }
 
-    .oe_stat_button, > div > button {
+    .oe_stat_button, 
+    div > .oe_stat_button, 
+    .oe_stat_button > button {
         border-radius: 0;
     }
 
-    .oe_stat_button:first-child, div:first-child > button {
+    > .oe_stat_button:first-child, 
+    > div:first-child > .oe_stat_button {
         @include border-start-radius($border-radius);
     }
 
-    .oe_stat_button:last-child, div:last-child > button {
+    > .oe_stat_button:last-child, 
```

Comment:
Nitpicking: this line adds a border radius to the container of the "More" button too. Maybe you can add `button` to this selector line so we don't have two border radiuses applied